presto-docs/src/main/sphinx/release/release-0.229.rst

```diff
@@ -0,0 +1,36 @@
+=============
+Release 0.229
+=============
+
+General Changes
+_______________
+* Fix an issue that would cause query failure when calling :func:`geometry_to_bing_tiles` on certain degenerate geometries.
+* Add geospatial function :func:`line_interpolate_point`.
+* Add support for ``CREATE FUNCTION``.
+* Add support for using configuration property ``stage.max-tasks-per-stage`` to limit the number of tasks for scan.
```

Comment:
`Add configuration property ...`

Reply:
The configuration property already exists and is being used. We're respecting its value in a different circumstance.

Reply:
Maybe `Add support to respect xxx` then?

Reply:
Changed to 
```
Add support to respect configuration property ``stage.max-tasks-per-stage`` for limiting the number of tasks per scan.
```